infra/blueprint-test/examples/simple_tf_module_with_test/test/setup/main.tf

```diff
@@ -16,7 +16,7 @@
 
 module "project" {
   source  = "terraform-google-modules/project-factory/google"
-  version = "~> 11.0"
```

Comment:
Since this is test data, we could update renovate to skip these

Reply:
@bharathkkb - Are you thinking to skip all TF for this repo `ignoreDeps`, or add this to `ignorePaths`?